keyboards/sawnsprojects/re65/keymaps/via/keymap.c

```diff
@@ -0,0 +1,95 @@
+// Copyright 2023 MaiTheSan (@maithesan)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+enum {
+    _BASE,
+    _FN1,
+    _FN2,
+    _FN3
```

Comment:
```suggestion
    _FN2,
    _FN3
non-functional layers removed
```suggestion
    _FN2,
    _FN3
```